### ISSUE TITLE
Access & Embargo link gives "Something went wrong" on Images

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/alexandria-v2#462]
When I log in as an administrator (brian) and view a single image object, then click on the 'Access & Embargo' link, I get an error.

**Comment by jcoyne** [comment]
Backtrace:
```
NoMethodError (undefined method `scan' for nil:NilClass):
  app/utils/identifier.rb:3:in `treeify'
  app/controllers/access_controller.rb:35:in `load_curation_concern'
```

**Comment by jcoyne** [comment]
Closed by 71fb346
